kahuna/public/js/components/gr-title/gr-title.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import 'titip';
+
+export const module = angular.module('grTitle', []);
+
+module.directive('grTitle', [function () {
```

Comment:
I always found the `title` attribute name confusing, so I might be biased against it :-)